src/app/lib/config/services/naidheachdan.js

```diff
@@ -16,22 +16,23 @@ import 'moment/locale/gd';
 export const service = {
   default: {
     lang: `gd`,
-    articleAuthor: `https://www.facebook.com/BBCNaidheachdan`,
-    articleTimestampPrefix: 'Updated',
+    articleAuthor: `https://www.facebook.com/bbcnews`,
+    articleTimestampPrefix: 'Ùraichte',
     atiAnalyticsAppName: 'news-naidheachdan',
     atiAnalyticsProducerId: '79',
-    brandName: "Naidheachdan a' BhBC",
-    product: "Naidheachdan a' BhBC",
+    brandName: 'BBC Naidheachdan',
+    product: 'BBC News',
+    serviceLocalizedName: 'Naidheachdan',
     defaultImage:
       'https://www.bbc.co.uk/news/special/2015/newsspec_11063/naidheachdan_1024x576.png',
-    defaultImageAltText: "Naidheachdan a' BhBC",
+    defaultImageAltText: 'BBC Naidheachdan',
     dir: `ltr`,
-    externalLinkText: ', external',
-    imageCaptionOffscreenText: 'Image caption, ',
-    videoCaptionOffscreenText: 'Video caption, ',
-    audioCaptionOffscreenText: 'Audio caption',
-    defaultCaptionOffscreenText: 'Caption, ',
-    imageCopyrightOffscreenText: 'Image source, ',
+    externalLinkText: ', taobh a-muigh',
+    imageCaptionOffscreenText: 'Tiotal an deilbh, ',
+    videoCaptionOffscreenText: 'Tiotal an deilbh, ',
```

Comment:
Are these meant to be the same?

Reply:
Good spot.  They are the same in the translations we've been sent - I've emailed our Gaelic translator to confirm.

Reply:
Fixed!